argilla-server/src/argilla_server/contexts/accounts.py

```diff
@@ -128,14 +129,14 @@ async def create_user(db: AsyncSession, user_attrs: dict, workspaces: Union[List
 
             await WorkspaceUser.create(
                 db,
-                workspace_id=workspace.id,
-                user_id=user.id,
+                workspace=workspace,
+                user=new_user,
```

Comment:
Reverting this change since it is failing

```suggestion
                workspace_id=workspace.id,
                user_id=new_user.id,
```